### PR TITLE
Attempt to fix travis again

### PR DESCRIPTION
**Pull request by rynowak**

### CODE REVIEW

File: test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/BuildIntegrationTest.cs [hunk 1]
```diff
@@ -19,7 +20,12 @@ public class BuildIntegrationTest : MSBuildIntegrationTestBase
             Assert.FileExists(result, OutputPath, "SimpleMvc.dll");
             Assert.FileExists(result, OutputPath, "SimpleMvc.PrecompiledViews.dll");
 
-            Assert.BuildOutputContainsLine(result, $"SimpleMvc -> {Path.Combine(Path.GetFullPath(Project.DirectoryPath), OutputPath, "SimpleMvc.PrecompiledViews.dll")}");
+            if (RuntimeEnvironment.OperatingSystemPlatform == Platform.Darwin)
```
Comment (pranavkm) [inline test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/BuildIntegrationTest.cs]: Did you mean `!= Darwin`? Also we have `ConditionalFacts` you could use to skip this test on OSX

Comment (rynowak) [reply test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/BuildIntegrationTest.cs]: I don't want to skip the whole test, just this one verification.

Comment (rynowak) [reply test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/BuildIntegrationTest.cs]: Yeah turning it into a `!=` oops 😆

**Pull request merged**